test/powershell/engine/COM/COM.Basic.Tests.ps1

```diff
@@ -49,6 +49,13 @@ try {
             $element | Should -Be $drives.Item($element.DriveLetter)
         }
 
+        It "Should be able to enumerate 'IADsMembers' object" {
+            $group = [ADSI]"WinNT://./Users,Group"
+            $members = $group.Invoke('Members')
+            $names = $members | ForEach-Object { $_.GetType().InvokeMember('Name', 'GetProperty', $null, $_, $null) }
+            $names | Should -Not -BeNullOrEmpty
```

Comment:
Are there any specific names we can validate against that are always expected for this COM object?

Reply:
I'm guess 'Authenticated Users' and 'INTERACTIVE'. Let me update and test it out.

Reply:
Test updated.